app/_hub/SmartParkingTechnology/google-logging/index.md

```diff
@@ -0,0 +1,121 @@
+---
+name: Kong Google Cloud Logging
+publisher: SmartParkingTechnology
+
+categories:
+  - logging
+
+type: plugin
+
+desc: Logs Kong requests with Google Cloud Logging
+
+description: |
+  Plugin for exporting Kong request data such as Service name, Route name, Consumer name or request latency to Google Cloud Logging.
+
+support_url: https://github.com/SmartParkingTechnology/kong-google-logging-plugin/issues
+
+source_url: https://github.com/SmartParkingTechnology/kong-google-logging-plugin
+
+license_type: Apache-2.0
+
+kong_version_compatibility:
+  community_edition:
+    compatible:
+      - 2.1.x
+
+params:
+  name: google-logging
+  api_id: true
+  service_id: false
+  consumer_id: false
```

Comment:
```suggestion
  consumer_id: true
```

Reply:
So far we only used and tested the plugin globally, I am not sure if it works for consumer and service (even though I can imagine it will).

Reply:
> So far we only used and tested the plugin globally, I am not sure if it works for consumer and service (even though I can imagine it will).

Ok thank you for your response. Please let us know after you have tested for Service, Route, Consumer.

Reply:
I am currently quite busy, could we do that as future work? I think the main use-case is anyway to run the plugin globally.

Reply:
> I am currently quite busy, could we do that as future work? I think the main use-case is anyway to run the plugin globally.

Absolutely.